stencil.config.ts

```diff
@@ -30,6 +30,7 @@ export const config: Config = {
   commonjs: {
     namedExports: {
       'stellar-sdk': [
+        'Server',
```

Comment:
What is this stuff? Its looks like we have to hard-code the pieces of our stellar-sdk we use, which feels weird/wrong. If `Server` is not included, can I access `Server` in our JS code? I guess I don't know what this config file is.

Reply:
Long story short Stencil needs you to explicitly mark (_some?_) what classes you're exporting from modules you've imported.
Something to do with rollout but I'm not an expert.

I do know that it is what's making things function currently.

Reply:
I almost missed your explanation! Let the person asking the question figure out if its resolved.

And ok sounds good, don't fix whats not broken. But this does seem a little sus